app/Models/AssetMaintenance.php

```diff
@@ -52,7 +52,8 @@ public static function getImprovementOptions()
         return [
             trans('admin/asset_maintenances/general.maintenance') => trans('admin/asset_maintenances/general.maintenance'),
             trans('admin/asset_maintenances/general.repair')      => trans('admin/asset_maintenances/general.repair'),
-            trans('admin/asset_maintenances/general.upgrade')     => trans('admin/asset_maintenances/general.upgrade')
+            trans('admin/asset_maintenances/general.upgrade')     => trans('admin/asset_maintenances/general.upgrade'),
+            'PAT test'      => 'PAT test',
```

Comment:
What's `PAT test` - and this array of things that just point to themselves feels funny. I don't even know what it does but it doesn't feel like it should be necessary. Maybe I'll figure out what it does in a bit.

Reply:
They don't point to themselves, it just builds the dropdown list of asset maintenance types. Someone asked for PAT testing, so I added it.